src/shared/components/svg-icon.js

```diff
@@ -34,10 +51,12 @@ export default function SvgIcon({
   }
   const markup = { __html: iconRegistry[name] };
 
-  const element = useRef();
+  const element = /** @type {Ref<HTMLElement>} */ (useRef());
   useLayoutEffect(() => {
     const svg = element.current.querySelector('svg');
-    svg.setAttribute('class', className);
+    if (svg) {
+      svg.setAttribute('class', className);
+    }
```

Comment:
The `svg` variable _should_ always be truthy here, but nothing in the code actually guarantees that.

Reply:
Does this warrant a comment? I could see a future dev being like...????